actioncable.go

```diff
@@ -362,6 +408,7 @@ type Command struct {
 type CommandIdentifier struct {
 	// Channel is the name of the channel.
 	Channel string
+	RawData json.RawMessage
```

Comment:
I don't need it public for my use case, but it is possible that the same way the client needs to add data there for the server, the server could do this for the client too and the client (our code) would want that.

But I'm fine making this private until a use case is found.